### ISSUE TITLE
About terms["nll"]

### ISSUE THREAD

**Issue by lgs00** [bloc97/CrossAttentionControl#27]
Thanks for your great work. In line 633 of gaussian_diffusion.py, terms["nll"] is calculated but not used. Whther it is a mistake, or whether it doesn't work.
` terms["nll"] = self._token_discrete_loss(model_out_x_start, get_logits, input_ids_x, mask=input_ids_mask, truncate=True, t=t) `
`terms["loss"] = terms["mse"] + decoder_nll + tT_loss`
